odoo/tools/populate.py

```diff
@@ -35,6 +35,14 @@ def root_factory():
 def randomize(vals, weights=None, seed=False, formatter=format_str, counter_offset=0):
     """ Return a factory for an iterator of values dicts with pseudo-randomly
     chosen values (among ``vals``) for a field.
+
+    :param list vals: list in which a value will be chosen, depending on `weights`
+    :param list weights: list of probabilistic weights
+    :param seed: optional initialization of the random number generator
+    :param function formatter: (val, counter, values) --> formatted_value
+    :param int counter_offset:
+    :returns: function of the form (iterator, field_name, model_name) -> values
+    :rtype: `function` (`iterator`, :class:`str`, :class:`str`) --> :class:`dict`
```

Comment:
Doesn't sphinx automatically format rtype? Also should not use single backticks in rst.

Reply:
Only if he finds a documentation link for a given term (and if he likes the way you organize the rtype content).  I prefer to sometimes enforce myself the same display than the `:type:`
![image](https://user-images.githubusercontent.com/19364378/81415511-d5de2600-9148-11ea-8b9c-3c336554cf6f.png)

Reply:
This is quite noisy IMHO